<launch>

    <node name="uav_com" pkg="uavcom" type="uav_com" output="screen" ns="uav1"/>
    <node name="uav_com" pkg="uavcom" type="uav_com" output="screen" ns="uav2"/>
    <node name="uav_com" pkg="uavcom" type="uav_com" output="screen" ns="uav3"/>
	
</launch>
